app/game_manager.py

```diff
@@ -22,14 +26,82 @@ def __init__(self):
         self.game_id = None  # the game lobby id that the server will provide for online multiplayer
         self.server_ip = None
         self.chess_board = None
+        self.term = Terminal()
 
     def create_lobby(self) -> int:
         """Used to create a game lobby on the server or locally."""
         pass
 
     def show_welcome_screen(self) -> None:
         """Prints startup screen."""
-        pass
+        print(self.term.home + self.term.white_on_black + self.term.clear)
```

Comment:
use fullscreen() context manager 
```suggestion
        with self.term.fullscreen():
            print(self.term.home + self.term.white_on_black + self.term.clear)
```
the whole function body need to be under `with` clause